src/js/stores/MesosStateStore.js

```diff
@@ -227,7 +227,7 @@ class MesosStateStore extends GetSetBaseStore {
   }
 
   getTasksByService(service) {
-    const { frameworks, tasks = [] } = this.get("lastMesosState");
+    const { frameworks = [], tasks = [] } = this.get("lastMesosState") || {};
```

Comment:
Here as well the find can fail on line 251.

Reply:
I can't see that one, frameworks is initialized  and framework has a guard clause later on 😕